### ISSUE TITLE
‼️ NOTICE: Python [ERROR: constructs 3.0.1 has requirement jsii~=1.1.0, but you'll have jsii 1.3.1 which is incompatible.]

### ISSUE THREAD

**Issue by iliapolo** [aws/aws-cdk#7433]
Please add your +1 👍 to let us know you have encountered this
---

### Overview:

Installation of **python** `aws-cdk.<module>` version 1.33.0 fails with a conflicting `jsii` version.

#### Complete Error Message:

```console
pip install aws-cdk.core
Collecting aws-cdk.core
  Using cached aws_cdk.core-1.33.0-py3-none-any.whl (624 kB)
Collecting jsii~=1.3.0
  Using cached jsii-1.3.1-py3-none-any.whl (263 kB)
Collecting publication>=0.0.3
  Using cached publication-0.0.3-py2.py3-none-any.whl (7.7 kB)
Collecting aws-cdk.cx-api==1.33.0
  Using cached aws_cdk.cx_api-1.33.0-py3-none-any.whl (101 kB)
Collecting constructs<4.0.0,>=3.0.0
  Using cached constructs-3.0.1-py3-none-any.whl (75 kB)
Collecting aws-cdk.cloud-assembly-schema==1.33.0
  Using cached aws_cdk.cloud_assembly_schema-1.33.0-py3-none-any.whl (89 kB)
Collecting python-dateutil
  Using cached python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
Collecting cattrs~=1.0.0
  Using cached cattrs-1.0.0-py2.py3-none-any.whl (14 kB)
Collecting typing-extensions~=3.7.4
  Using cached typing_extensions-3.7.4.2-py3-none-any.whl (22 kB)
Collecting attrs~=19.3.0
  Using cached attrs-19.3.0-py2.py3-none-any.whl (39 kB)
Collecting six>=1.5
  Using cached six-1.14.0-py2.py3-none-any.whl (10 kB)
ERROR: constructs 3.0.1 has requirement jsii~=1.1.0, but you'll have jsii 1.3.1 which is incompatible.
Installing collected packages: six, python-dateutil, attrs, cattrs, typing-extensions, jsii, publication, aws-cdk.cloud-assembly-schema, aws-cdk.cx-api, constructs, aws-cdk.core
Successfully installed attrs-19.3.0 aws-cdk.cloud-assembly-schema-1.33.0 aws-cdk.core-1.33.0 aws-cdk.cx-api-1.33.0 cattrs-1.0.0 constructs-3.0.1 jsii-1.3.1 publication-0.0.3 python-dateutil-2.8.1 six-1.14.0 typing-extensions-3.7.4.2
```

### Workaround:

Current workaround is to use version `1.32.2` of `aws-cdk`.

### Related Issues:

- https://github.com/aws/aws-cdk/issues/7426

**Comment by eladb** [comment]
We are going to release a patch which reverts the upgrade to jsii 1.3.1 to work around this issue.

**Comment by eladb** [comment]
Patch release: https://github.com/aws/aws-cdk/pull/7437

**Comment by eladb** [comment]
Released v1.33.1 which resolves the issue
